### ISSUE TITLE
Conflict with Iris, view bobbing no longer works.

### ISSUE THREAD

**Issue by Slayer5934** [Emafire003/LightWithin#15]
**Describe the bug**
With Iris and Light Within installed, view bobbing no longer works.

**Version information**
LightWithin mod version: <0.3.1>
Minecraft Version: <1.19.2>
Fabric API version: <0.77.0>

**Additional context**
I found this after extensive testing for my modpack Tekxit, after narrowing it down with 4 mods in the folder (fabric api, iris, light within, sodium) I found that removing either Iris or Light Within makes view bobbing work again.

**Comment by Slayer5934** [comment]
https://github.com/IrisShaders/Iris/issues/2160

**Comment by Slayer5934** [comment]
(forgot to mention shaders must be enabled)

**Comment by Emafire003** [comment]
That's quite weird. Anyway, it is apparently already fixed in the version i'm working on (1.1.0, mc1.20.1). I suspect it is linked to a problem with Renderer a library used internally by LightWithin to render stuff on the HUD. 
I was thinking about porting this version (1.1.0) to 1.19.2 for modpacks, but it will take a while, since there is like a 90+ commits difference between the two versions. I will be checking if this issue presents itself again on 1.19.2.

**Comment by Emafire003** [comment]
Yes, it's also already fixed in 1.19.2. I will be publishing the 1.1.0+1.19.2 version shortly. Let me know if you need a copy for testing before the release, i will make a beta-release on github.
